feature/authentication/presentation/src/main/java/com/pob/umpa/feature/authentication/presentation/SignActivity.kt

```
@@ -23,21 +26,25 @@ class SignActivity : ComponentActivity() {
-            val navController = rememberNavController()
-            val mainNavigator = object : Navigator {
-                override fun navigateToMainActivity(context: Context) {}
-            }
-            SignInScreen(navController, mainNavigator)
+@Preview(showBackground = true)
+@Composable
+fun GreetingPreview() {
+    UmpaTheme {
+        val navController = rememberNavController()
+        val mainNavigator = object : Navigator {
+            override fun navigateToMainActivity(context: Context) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)